src/clait/format/templates/french_autoentrepreneur.py

```diff
@@ -177,7 +177,8 @@
 \ifthenelse{{\equal{{\FactureAcquittee}}{{paid}}}}{{
     Facture acquittée.
 }}{{
-    Facture à régler, par chèque ou par virement bancaire :
+    Paiement à 60 jours suivant réception de la facture par chèque ou virement
```

Comment:
I'd rather make it per default *30 jours fin de mois* (the default legal requirement, AFAIK) instead of *60 jours*